### ISSUE TITLE
the target dir not exists

### ISSUE THREAD

**Issue by OceanBelongsToMe** [tauri-apps/tauri#3044]
### **Describe the bug**
the target dir not exists

### **To Reproduce**
1. yarn create tauri-app
2. choose rust dominator item
3. yarn && yarn tauri build

### **Screenshots**
![image](https://user-images.githubusercontent.com/22573382/145146555-48913ba4-6af1-48e7-82a6-9647c4be5177.png)

### **Platform and Versions (required):**

```
➜  power-clip git:(master) ✗ yarn tauri info
yarn run v1.22.17
warning package.json: No license field
$ tauri info
 app:spawn Running "/Users/ocean/Documents/workplace/mystudywork/rust/project/power-clip/node_modules/@tauri-apps/cli/bin/tauri-cli tauri info" +0ms


Operating System - Mac OS, version 11.6.1 X64

Node.js environment
  Node.js - 16.13.0
  @tauri-apps/cli - 1.0.0-beta.10
  @tauri-apps/api - 1.0.0-beta.8

Global packages
  npm - 8.1.0
  yarn - 1.22.17

Rust environment
  rustc - 1.56.1
  cargo - 1.56.0

App directory structure
/dist
/target
/node_modules
/src-tauri
/.git
/.idea
/src

App
  tauri.rs - 1.0.0-beta.8 (no lockfile)
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - ../dist
  devPath - http://localhost:10001/
  bundler - Rollup

✨  Done in 19.45s.
```

**Comment by OceanBelongsToMe** [comment]
All right, My current solution is to add .cargo/config.toml in src-tauri ,and .cargo/config.toml file add content ,like as:
```
[build]
targe-dir="..." # absolute path , i do not know how to use relative one :)
```

**Comment by lucasfernog** [comment]
Where is your target dir in this setup? I assume you have a workspace right? Maybe our code isn't finding it properly.

**Comment by OceanBelongsToMe** [comment]
https://github.com/tauri-apps/tauri/blob/c664e9dbe012faab462a719ffd00bdf80c321b75/tooling/cli.rs/src/build.rs#L157
I read this code，and My understanding is the code will return target path in the src-tauri dir，but the real target path which has Compile files is src in the project root

**Comment by lucasfernog** [comment]
it should read the workspace, i'll take a look this week.

**Comment by lucasfernog** [comment]
This issue was fixed by #2644. It will be available on the next release. If you want to use it right now, you can install cli.rs as a Cargo subcommand:

```
# install
$ cargo install --git https://github.com/tauri-apps/tauri/ --branch next
# then build your app
$ cargo tauri build
```
